curriculum/challenges/english/blocks/workshop-bookstore-page/68ec6cd5b7e8f5a8f7319e32.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 68ec6cd5b7e8f5a8f7319e32
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+The `<div>` element is used as a container to group other HTML elements. You will mainly use the `<div>` element when you want to group HTML elements that will share a set of CSS styles.
+
+Below the `<p>` element, add a `<div>` element. This div will be a container for your book cards.
+
+# --hints--
+
+Your `div` element should have an opening tag. Opening tags have this syntax: `<elementName>`.
+
+```js
+assert.exists(document.querySelector('div'));
+```
+
+Your `div` element should have a closing tag. Closing tags have a `/` and the same element name: `</div>`.
+
+```js
+assert.match(code, /<\/div\s*>/);
+```
```

Comment:
see my note earlier on why these tests are not needed.
we just need one test to make sure the `div `element exists